src/screens/index.tsx

```diff
@@ -375,12 +375,14 @@ function Blocks() {
       >
         {virtualizer.getVirtualItems().map(({ key, index, size, start }) => {
           const { block, status } = blocks[index] || {}
+          const isPending = status === 'pending'
           if (!block) return
           return (
             <Link
```

Comment:
Could be worth just conditionally rendering the `Link` component here so the anchor element doesn't leak its way into the DOM.

Example:
```ts
const wrap = (children: React.ReactElement) =>
  status === 'mined' ? (
    <Link
      onClick={() => setPosition(parentRef.current?.scrollTop!)}
      to={`block/${block.number}`}
      key={key}
    >
      {children}
    </Link>
  ) : (
    children
  )
// ...
return wrap(
  // ...
)
```

LMK what you think!

Reply:
Actually, just realised that this might be a bug as we want to show the pending block details and possible transactions. However it seems that Anvil is returning zero data for fetching a pending **block number**. If we fetch the block by `"pending"` tag instead, then we can retrieve the pending block details (I'm guessing this could be an Anvil bug).

I think the best way forward would be:
1. For **mined** blocks, we use `blockNumber` in `useBlock` ([as seen here](https://github.com/paradigmxyz/rivet/blob/main/src/screens/block-details.tsx#L29)),
2. For **pending** blocks, we should use `blockTag: "pending"` instead as an argument to `useBlock` – we will probably need to pass a query param to the screen so we can disciminate here.

Reply:
12fa7b2929abf92a39af5c7427b8929ccb1214e9

Reply:
But probably it would be great to fix it on Anvil level since there could be more people that experience this bug